test/classes/engin/callSuperMethods.chpl

```diff
@@ -0,0 +1,31 @@
+class A {
+
+  proc foo(){
+    writeln("A.foo() called");
+  }
+}
+
+class B:A {
+
+  /*
+      bar runs if there is no foo that calls super.foo
+
+  proc bar() {
+    super.foo();
+    writeln("B.foo() called");
+  }
+  */
+  proc foo() {
```

Comment:
My strategy is generally to make a few test cases for error messages - show the ones that work either in individual tests, or all in one test file that is separate from the error; and put each of the ways to trigger the error into their own file, so we can easily see if all have been fixed in one go or not.